.gitmodules

```diff
@@ -25,3 +25,6 @@
 	path = lib/openzeppelin-contracts-upgradeable
 	url = https://github.com/openzeppelin/openzeppelin-contracts-upgradeable
 	branch = v4.8.0
+[submodule "lib/murky"]
+	path = lib/murky
+	url = https://github.com/dmfxyz/murky
```

Comment:
I see code is not audited. Could we use [OZ's Merkle Proof](https://docs.openzeppelin.com/contracts/4.x/api/utils#MerkleProof) ?

Reply:
this is just for foundry utils to generate the proof offchain, therefore it doesn't matter if the code is potentially insecure. Verification does indeed use OZs audited lib.